### PR TITLE
[shopsys] added convertim package

### PR DESCRIPTION
**Pull request by grossmannmartin**
#### Description, the reason for the PR

preparation for a [Convertim](https://www.convertim.com) integration.

#### Fixes issues
... 

#### Check the appropriate checkboxes below, please

- [ ] [BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/) 
- [ ] New feature 
- [x] I have read and signed [License Agreement for contributions](https://www.shopsys.com/license-agreement)

## Summary by CodeRabbit

- **New Features**
	- Introduced a GitHub Actions workflow for automated checks and tests on code changes.
	- Added a README.md file detailing the integration of the Convertim one-click checkout feature.
	- Implemented a CONTRIBUTING.md file with guidelines for community contributions.
	- Created a LICENSE file specifying the MIT License terms for software distribution.
	- Integrated the Convertim package into the project to enhance conversion-related features.
	- Added service configurations for the ShopsysConvertimBundle to streamline service management.

- **Documentation**
	- Added comprehensive documentation in the README.md and CONTRIBUTING.md files.

- **Chores**
	- Introduced a .gitignore file to maintain a cleaner repository by excluding unnecessary files.
	- Added composer.json and phpstan.neon files for package configuration and static analysis setup.
	- Updated packages_registry.yaml to accommodate new packages.

----
:globe_with_meridians: Live Preview:
  - https://convertim.odin.shopsys.cloud
  - https://cz.convertim.odin.shopsys.cloud

### CODE REVIEW

File: docs/contributing/guidelines-for-writing-upgrade.md [hunk 1]
```diff
@@ -50,7 +50,8 @@ Each upgrade file must have a link to the main UPGRADE.md file with general info
 -   shopsys/product-feed-zbozi
 -   shopsys/product-feed-luigis-box
 -   shopsys/article-feed-luigis-box
--   shopsys/luigis-box
```
Comment (TomasLudvik) [inline docs/contributing/guidelines-for-writing-upgrade.md:53]: This should not be removed, right?

File: packages/convertim/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/.php_cs.cache
```
Comment (TomasLudvik) [inline packages/convertim/.gitignore:2]: We have also `/.idea` in most of our packages

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: docs/contributing/guidelines-for-writing-upgrade.md (hunk 1 above)
Comment (grossmannmartin) [reply docs/contributing/guidelines-for-writing-upgrade.md:53]: no, it's a mistake. thanks for noticing! - 1b70a122c6

File: packages/convertim/.gitignore (hunk 2 above)
Comment (grossmannmartin) [reply packages/convertim/.gitignore:2]: added – 66b953cb0f

**Review by TomasLudvik — APPROVED** [review]

**Pull request merged**